### ISSUE TITLE
Ctrl+V with audio file on clipboard should import it

### ISSUE THREAD

**Issue by saintmatthieu** [audacity/audacity#6028]
If I copy an audio file on my computer's clipboard, ctrl+V on Audacity should import it.

**Comment by DaviFN** [comment]
If an audio track is copied within the program, CTRL + V pastes it; in such circunstance, the proposed behavior would conflict with this already existing behavior.

**Comment by LWinterberg** [comment]
If an audio track is copied within Audacity, there is no audio file in the clipboard so the proposed new behavior wouldn't trigger :)

**Comment by DaviFN** [comment]
> If an audio track is copied within Audacity, there is no audio file in the clipboard so the proposed new behavior wouldn't trigger :)

**Comment by DaviFN** [comment]
Nope; it is still possible to paste the file using explorer after having copied an audio clip within Audacity.

**Comment by saintmatthieu** [comment]
QA (copy to PR when ready):
- [ ] Paste order of tracks respects copy order of files
- [ ] Copy a track, then copy a wav file, paste: the wav file is pasted. Vice versa.

**Comment by saintmatthieu** [comment]
Thanks @dozzzzer !
The QA checkboxes on the PR aren't checked, have you tested these items, too ?

**Comment by dozzzzer** [comment]
Now I have :)
And I found one minor problem (on Windows and Linux only): when importing of one of the files fails, the entire operation isn't aborted, instead, all the files before the failed file's position on the list get imported.

**Comment by saintmatthieu** [comment]
This isn't a regression introduced by this ticket, and the implementation of this behaviour (one failing import reverts project state) isn't trivial, so I'll leave it out. (Maybe a ticket exists already?)
I considered not having the multiple-import action interrupted, trying to import as many files as possible, but that can get annoying for the user, because each failing import comes with one dialog or two, so the user might have to repeatedly close dialogs.
However, I noticed that if one import fails, the tempo detection is not run on the previous successful imports. Added a correction for that.
